README.md

```diff
@@ -79,6 +79,7 @@ Flags:
       --reboot-sentinel string              path to file whose existence signals need to reboot (default "/var/run/reboot-required")
       --slack-hook-url string               slack hook URL for reboot notfications
       --slack-username string               slack username for reboot notfications (default "kured")
+      --days-to-exclude []string    List of days where rebooting should be disabled e.g. 'Sunday, Monday
```

Comment:
Excluding days is nice but it would be nicer if we could also exclude time windows. For exemple it would be nice if  production clusters were not rebooted at night and trigger on call shift alerts.